src/spaces/grid_general.jl

```diff
@@ -69,14 +71,14 @@ function calculate_hyperrectangle(space::AbstractGridSpace{D,P}, r) where {D,P}
     space_size = spacesize(space)
     r_notover = [p_d ? r < s_d ÷ 2 : r < s_d for (p_d, s_d) in zip(P, space_size)]
     if all(r_notover)
-        hyperrect = Iterators.product((-r:r for _ in 1:D)...) 
+        hyperrect = Iterators.product((-r:r for _ in 1:D)...)
     else
         odd_s, half_s = space_size .% 2, space_size .÷ 2
         r_dims_P = min.(r, half_s)
         r_dims_notP = min.(r, space_size)
-        from_to = (P[i] ? 
-                    (-r_dims_P[i]:r_dims_P[i]-(r_dims_P[i] == half_s[i] && odd_s[i] == 0)) : 
-                    (-r_dims_notP[i]:r_dims_notP[i]) for i in 1:D)
+        from_to = (P[i] ?
+                   (-r_dims_P[i]:r_dims_P[i]-(r_dims_P[i]==half_s[i]&&odd_s[i]==0)) :
+                   (-r_dims_notP[i]:r_dims_notP[i]) for i in 1:D)
```

Comment:
Can you please remove all formatting changes you did in this PR? I also find them unhelpful: removing all these spaces makes the code less readable.

Reply:
Oops, did not mean to change that.

Reply:
They should be gone now.